tesseract_rosutils/src/conversions.cpp

```diff
@@ -142,4 +143,34 @@ std::vector<tesseract_msgs::JointState> trajectoryFromCSVFile(const std::string&
   return trajectory;
 }
 
+trajectory_msgs::JointTrajectory toRosJointTrajectory(tesseract_common::JointTrajectory joint_trajectory, tesseract_common::JointState initial_state){
```

Comment:
@Levi-Armstrong using the `EnvState` seems to confuse things in our application since the human model is represented in the environment as well, but we don't want the human's position represented in the robot trajectory. How would you recommend just getting the robot's initial state?